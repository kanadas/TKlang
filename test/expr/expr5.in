(\a b -> if a + b == /*tralalalaal*/0 then "Zero" else a + b) 0 0
