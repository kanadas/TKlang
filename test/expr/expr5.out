expr/expr5.in
Type mismatch: [bool] and [int] in expression:
if a + b == 0 then true else a + b
[EIf (ERel (EAdd (EIdent (Ident {getIdent = "a"})) (EIdent (Ident {getIdent = "b"}))) (RelOp "==") (EInt 0)) ETrue (EAdd (EIdent (Ident {getIdent = "a"})) (EIdent (Ident {getIdent = "b"})))]
